packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -160,8 +161,28 @@ export default class ElectronAppWrapper {
 			this.win_.setPosition(primaryDisplayWidth / 2 - windowWidth, primaryDisplayHeight / 2 - windowHeight);
 		}
 
-		this.win_.webContents.on('unresponsive', async () => {
-			await this.handleAppFailure(_('Window unresponsive.'), true);
+		let unresponsiveTimeout: ReturnType<typeof setTimeout>|null = null;
+
+		this.win_.webContents.on('unresponsive', () => {
+			// Don't show the "unresponsive" dialog immediately -- the "unresponsive" event
```

Comment:
It might make sense to include a log statement here.

As the `Logger` is created in the renderer process, this would likely need to be a `console.log`.